__tests__/app.test.js

```diff
@@ -24,3 +24,27 @@ describe('GET /api/topics', () => {
     });
 });
 
+
+describe('GET /api/articles', () => {
+    test('200: responds with an array of all articles', () => {
+        return request(app)
+            .get('/api/articles')
+            .expect(200)
+            .then(({ body }) => {
+                const { articles } = body;
+                for (const article of articles) {
```

Comment:
Need to check that the length of the articles array here - could be potentially pass by default with an empty array

Reply:
okay. my bad. will include that now